script-library/common-alpine.sh

```diff
@@ -184,49 +184,66 @@ chmod +x /usr/local/bin/code
+    local gitbranch='`export BRANCH=$(git describe --contains --all HEAD 2>/dev/null); \
+        if [ "${BRANCH}" != "" ]; then \
+            echo -n "\[\033[0;36m\](\[\033[1;31m\]${BRANCH}" \
+            && if git ls-files --error-unmatch -m --directory --no-empty-directory -o --exclude-standard ":/*" > /dev/null 2>&1; then \
+                    echo -n " \[\033[1;33m\]✗"; \
+            fi \
+            && echo -n "\[\033[0;36m\]) "; \
+        fi`'
+    local lightblue='\[\033[1;34m\]'
+    local removecolor='\[\033[0m\]'
+    PS1="${userpart} ${lightblue}\w ${gitbranch}${removecolor}\$ "
+    unset -f __bash_prompt
 }
-SCM_THEME_PROMPT_PREFIX="\${reset_color}\${cyan}(\${bold_red}"
-SCM_THEME_PROMPT_SUFFIX="\${reset_color} "
-SCM_THEME_PROMPT_DIRTY=" \${bold_yellow}✗\${reset_color}\${cyan})"
-SCM_THEME_PROMPT_CLEAN="\${reset_color}\${cyan})"
-SCM_GIT_SHOW_MINIMAL_INFO="true"
-safe_append_prompt_command prompt
+__bash_prompt
 EOF
 )"
 CODESPACES_ZSH="$(cat \
-<<EOF
+<<'EOF'
 prompt() {
-    if [ ! -z "\${GITHUB_USER}" ]; then
-        local USERNAME="gh:@\${GITHUB_USER}"
+    if [ "${GITHUB_USER}" != "" ]; then
+        local USERNAME="@${GITHUB_USER}"
     else
-        local USERNAME="\$(whoami)"
+        local USERNAME="%n"
     fi
-    PROMPT="%{\$fg[green]%}\${USERNAME} %(?:%{\$reset_color%}➜ :%{\$fg_bold[red]%}➜ )"
-    PROMPT+='%{\$fg_bold[blue]%}%~%{\$reset_color%} \$(git_prompt_info)%{\$fg[white]%}$ %{\$reset_color%}'
+    PROMPT="%{$fg[green]%}${USERNAME} %(?:%{$reset_color%}➜ :%{$fg_bold[red]%}➜ )"
+    PROMPT+='%{$fg_bold[blue]%}%~%{$reset_color%} $(git_prompt_info)%{$fg[white]%}$ %{$reset_color%}'
 }
-ZSH_THEME_GIT_PROMPT_PREFIX="%{\$fg_bold[cyan]%}(%{\$fg_bold[red]%}"
-ZSH_THEME_GIT_PROMPT_SUFFIX="%{\$reset_color%} "
-ZSH_THEME_GIT_PROMPT_DIRTY=" %{\$fg_bold[yellow]%}✗%{\$fg_bold[cyan]%})"
-ZSH_THEME_GIT_PROMPT_CLEAN="%{\$fg_bold[cyan]%})"
+ZSH_THEME_GIT_PROMPT_PREFIX="%{$fg_bold[cyan]%}(%{$fg_bold[red]%}"
+ZSH_THEME_GIT_PROMPT_SUFFIX="%{$reset_color%} "
+ZSH_THEME_GIT_PROMPT_DIRTY=" %{$fg_bold[yellow]%}✗%{$fg_bold[cyan]%})"
+ZSH_THEME_GIT_PROMPT_CLEAN="%{$fg_bold[cyan]%})"
 prompt
 EOF
 )"
 
+# Add notice that Oh My Bash! has been removed from images and how to provide information on how to install manually
+OMB_README="$(cat \
+<<'EOF'
+"Oh My Bash!" has been removed from this image in favor of a simple shell prompt. If you still
+wish to use it, remove "~/.oh-my-bash" and install it from: https://github.com/ohmybash/oh-my-bash
+You may also want to consider "Bash-it" as an alternative: https://github.com/bash-it/bash-it
+See https://github.com/microsoft/vscode-dev-containers/issues/674#issuecomment-783474956
+EOF
+)"
+OMB_STUB="$(cat \
+<<'EOF'
+#!/usr/bin/env bash
+cd "$(dirname $0)"
+if [ -t 1 ]; then
+    cat README.md
+fi
+EOF
+)"
+
+
 # Adapted Oh My Zsh! install step to work with both "Oh Mys" rather than relying on an installer script
```

Comment:
Does this comment need to change now that OMB is being removed?

Reply:
We could remove it out of a function to simplify, but I opted to leave it as-is to reduce the amount of change in the PR. The Oh My Zsh! install still works fine with it, so it's not "must have" to change.

Reply:
I should also mention that I used a git clone directly here rather than the up-stream script since this reduces our supply chain risk a bit (we only have to consider the repository, not what the install script does). It's a supported install method for both OMB and OMZ.  OMB just had a slightly different spot for the template which is the bit I yanked.

Most of the other edits are adding quotes to things that technically should have been quoted to begin with... you just don't normally see spaces in these paths so it worked fine as it was before.

Reply:
As a part of another change, I went ahead and dropped the function and merged it in with the ZSH setup section at the bottom.